cloud-native-edition/pygluu/kubernetes/templates/helm/gluu/README.md

```diff
@@ -213,7 +214,7 @@ Kubernetes: `>=v1.22.0-0`
 | fido2.image.pullPolicy | string | `"IfNotPresent"` | Image pullPolicy to use for deploying. |
 | fido2.image.pullSecrets | list | `[]` | Image Pull Secrets |
 | fido2.image.repository | string | `"gluufederation/fido2"` | Image  to use for deploying. |
-| fido2.image.tag | string | `"4.5.5-1"` | Image  tag to use for deploying. |
+| fido2.image.tag | string | `"4.5.5-2"` | Image  tag to use for deploying. |
```

Comment:
Tag should be `4.5.6-x` if this PR targeting for new release.

Reply:
Its not targeting the new release. We should open a separate PR for that